src/parsers/transformer.rs

```diff
@@ -237,9 +302,57 @@ fn fake_full_address() -> String {
     format!("{}, {}, {}", line_1, city_name, state)
 }
 
-fn fake_full_name() -> String {
-    let first: String = FirstName().fake();
-    let last: String = LastName().fake();
+fn fake_first_name(
+    value: &str,
+    args: &Option<HashMap<String, String>>,
+    id: Option<&str>,
+) -> String {
+    let deterministic = is_deterministic(args);
+
+    if deterministic && id.is_none() {
+        panic!("Deterministic FakeFirstName transformer requires 'id_column' argument to be set and valid");
```

Comment:
Can we validate this on a higher level eg in `strategy_structs.rs`? So that the strategy file can be checked as part of the `validate` function without needing to run the anonymisation process to get to this panic.

Reply:
I moved it in strategy.rs file as IIUc most validation logic is in strategies.rs file